src/main/java/academicweapon/ui/Ui.java

```diff
@@ -0,0 +1,113 @@
+package academicweapon.ui;
+
+import academicweapon.exceptions.DukeExceptions;
+import academicweapon.task.Task;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * The Ui class handles the user interface
+ */
+public class Ui {
+    /**
+     * Displays a welcome message when the application starts.
+     */
+    public void showWelcome() {
+        Ui.showLine();
+        System.out.println("Hello! I'm AcademicWeapon");
+        System.out.println("What can I do for you?");
+        Ui.showLine();
+    }
+
+    /**
+     * Displays a horizontal line as a separator in the console.
+     */
+    public static void showLine() {
+        System.out.println("____________________________________________________________");
```

Comment:
You can probably store this as a constant, which would make sure all lines are the same length.